src/project/src/test/java/PetTest.java

```diff
@@ -0,0 +1,109 @@
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class PetTest {
+
+    private Pet pet;
+    @BeforeEach
+    void setUp() {
+        pet = new Pet("Brown", 2, "Samojeed");
+    }
+
+    @Test
+    void canGetName() {
+        //Arrange
+        var name = "Berk";
```

Comment:
pet = new Pet("Brown", 2, "Samojeed"); die had in de arrange fase gemoeten. Om daarna in de act; string name = pet.getname() te doen en die te vergelijken met age van 2. assertEquals(2, age);